floss/main.py

```diff
@@ -142,6 +142,13 @@ def make_parser(argv):
         help="minimum string length",
     )
 
+    parser.add_argument(
+        "-L",
+        "--large-file",
+        action="store_true",
+        help="allow processing files larger than {} MB".format(int(MAX_FILE_SIZE / MEGABYTE))
```

Comment:
nice, please also add `if show_all_options else argparse.SUPPRESS` here so this shows up under the "advanced" options via `-H`

Reply:
Fixed in #568 🙂